src/app/components/homepage/index.ts

```diff
@@ -0,0 +1,59 @@
+const startButton = <HTMLElement>document.querySelector('#start-game-button')
+const characterButton = <HTMLInputElement>(
+  document.querySelector('#character-dropdown')
+)
+const settingButton = <HTMLElement>document.querySelector('#settings-button')
+const infoButton = <HTMLElement>document.querySelector('#info-button')
+const startScreen = <HTMLElement>document.querySelector('.start__screen')
+const gameScreen = <HTMLElement>document.querySelector('.game__screen')
+const startScreenLogo = <HTMLElement>(
+  document.querySelector('.start__screen__logo')
+)
+
+export class HomePage {
+  removeLogoClasses() {
+    startScreenLogo.classList.remove('.start__screen__logo--elfs')
+    startScreenLogo.classList.remove('start__screen__logo--people')
+    startScreenLogo.classList.remove('start__screen__logo--dwarfs')
+  }
+  addStartFunction(): void {
+    startButton.addEventListener('click', () => {
+      startScreen.classList.add('start__screen--opened')
+      startScreen.classList.remove('start__screen--closed')
+      setTimeout(() => {
+        gameScreen.style.zIndex = '0'
+      }, 500)
+    })
+  }
+
+  addCharacterFunction(): void {
+    characterButton.addEventListener('change', () => {
+      const value = characterButton.value
+      if (value === 'elfs') {
+        this.removeLogoClasses()
+        startScreenLogo.classList.add('start__screen__logo--elfs')
+      } else if (value === 'dwarfs') {
+        this.removeLogoClasses()
+        startScreenLogo.classList.add('start__screen__logo--dwarfs')
+      } else {
+        this.removeLogoClasses()
+        startScreenLogo.classList.add('start__screen__logo--people')
+      }
```

Comment:
Alright here.. add new class method changeLogo(value)
Inside add 
this.removeLogoClasses()
StartScreenLogo.classList.add('start__screen__logo--${value}') then instead of if statement here you will just replace it with this.changeLogo(value)